hieradata/class/jenkins.yaml

```diff
@@ -29,6 +29,11 @@ govuk_jenkins::config:
       --httpPort=$HTTP_PORT
       --ajp13Port=-1
       --sessionTimeout=1440
+  JAVA_ARGS:
+    value: >-
+      -Djava.awt.headless=true
```

Comment:
These first two lines are copied from whatever was in `JAVA_ARGS` before this change; is there any way to merge this change in, rather than simply override with a copy-and-paste? Or does it not matter?